### PR TITLE
Konnect August Release Notes

### PR DESCRIPTION
**Pull request by haydenlam**
### Summary
Kong Konnect August updates to the release notes.

### Reason
Monthly release notes.

### Testing
None required.

### CODE REVIEW

**Comment by haydenlam** [comment]
This PR replaces https://github.com/Kong/docs.konghq.com/pull/4324

File: app/konnect/updates.md [hunk 1]
```diff
@@ -10,6 +10,30 @@ an application that lets you manage configuration for multiple runtimes
...
+
+: {{site.konnect_short_name}} users can now generate personal access tokens (PATs). PATs are used as an alternative method of authentication for [decK](/deck/1.14.x/guides/konnect/) commands avoiding the need to use traditional username and passwords.
+
+**AWS Marketplace Listing**
+
+: {{site.konnect_short_name}} Enterprise can now be purchased through the [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-7zds3oxx3ntjy) or via private contract with your Amazon Web Services account team. This can greatly simplify the procuement process for existing AWS customers.
+
+**DevPortal Custom Domain Progress Indicator**
+
+: We've added an in interactive checklist when configuring a [custom domain](/konnect/dev-portal/customization/#custom-dev-portal-url) for your developer portal. Now you can get an indigation whether your DNS changes have propogated and when the SSL certificate has been generated by {{site.konnect_short_name}}.
```
Comment (mheap) [inline app/konnect/updates.md:31]:
```suggestion
: We've added an in interactive checklist when configuring a [custom domain](/konnect/dev-portal/customization/#custom-dev-portal-url) for your developer portal. There is now an indicator to help understand if your DNS changes have propogated, and when the SSL certificate has been generated by {{site.konnect_short_name}}.
```

File: app/konnect/updates.md [hunk 2]
```diff
@@ -10,6 +10,30 @@ an application that lets you manage configuration for multiple runtimes
...
+
+: {{site.konnect_short_name}} launches generic OIDC support to allow Single-Sign-On (SSO) for platform login. Customers now have the ability to configure a separate OIDC-compliant Identity Provider (IdP) for {{site.konnect_short_name}}. Generic OIDC is an Enterprise feature.
+
+**Personal Access Tokens**
+
+: {{site.konnect_short_name}} users can now generate personal access tokens (PATs). PATs are used as an alternative method of authentication for [decK](/deck/1.14.x/guides/konnect/) commands avoiding the need to use traditional username and passwords.
+
+**AWS Marketplace Listing**
+
+: {{site.konnect_short_name}} Enterprise can now be purchased through the [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-7zds3oxx3ntjy) or via private contract with your Amazon Web Services account team. This can greatly simplify the procuement process for existing AWS customers.
```
Comment (mheap) [inline app/konnect/updates.md:27]:
```suggestion
: {{site.konnect_short_name}} Enterprise can now be purchased through the [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-7zds3oxx3ntjy) or via private contract with your Amazon Web Services account team. This greatly simplifies the procurement process for existing AWS customers.
```

File: app/konnect/updates.md [hunk 3]
```diff
@@ -10,6 +10,30 @@ an application that lets you manage configuration for multiple runtimes
...
+
+**DevPortal Custom Domain Progress Indicator**
+
+: We've added an in interactive checklist when configuring a [custom domain](/konnect/dev-portal/customization/#custom-dev-portal-url) for your developer portal. Now you can get an indigation whether your DNS changes have propogated and when the SSL certificate has been generated by {{site.konnect_short_name}}.
+
+**Download option to all reports**
+
+: We have introduced an "Export" option to all reports inside the Overview page in Analytics. That will download an unpivoted CSV of the report to your local harddrive that can be used for offline analysis.
```
Comment (mheap) [inline app/konnect/updates.md:35]:
```suggestion
: We have introduced an "Export" option to all reports inside the overview page in Analytics. Exporting a report downloads an unpivoted CSV to your local machine that can be used for offline analysis.
```

**Review by mheap — APPROVED** [review]

**Pull request merged**